src/main/java/org/hmhb/organization/DefaultOrganizationService.java

```diff
@@ -59,9 +58,7 @@ public Organization getById(
     @Override
     public List<Organization> getAll() {
         LOGGER.debug("getAll called");
-        List<Organization> target = new ArrayList<>();
-        dao.findAll().forEach(target::add);
-        return target;
+        return dao.findAllByOrderByNameAsc();
```

Comment:
I added this so organizations would be ordered and it reduced the code a little bit.